dask_ml/model_selection/methods.py

```diff
@@ -314,16 +430,32 @@ def fit_and_score(
     params=None,
     fit_params=None,
     return_train_score=True,
+    sample_weight=None,
```

Comment:
Place at the end and make it `None` by default to make it "backward compatible".